app/internal/user.py

```diff
@@ -0,0 +1,48 @@
+from app.database import models, schemas
+from sqlalchemy.orm import Session
+from app.internal.security.ouath2 import pwd_context
+
+# pwd_context = CryptContext(schemes=["bcrypt"])
+
+
+def get_by_id(db: Session, user_id: int) -> models.User:
```

Comment:
Prefer Depends

Reply:
Same as above, regarding the ability to use Depends in a regular function and not in a route.